libs/ledger-live-common/src/families/cosmos/logic.ts

```diff
@@ -215,3 +215,7 @@ export function getRedelegationCompletionDate(
   const currentRedelegation = getRedelegation(account, delegation);
   return currentRedelegation ? currentRedelegation.completionDate : null;
 }
+
+export const parseUatomStrAsAtomNumber = (uatoms: string): number => {
+  return parseFloat(uatoms) / 1000000.0;
```

Comment:
why `1000000.0` ?

Reply:
I think it is OK here. Since the name of the method "parseUatomStrAsAtomNumber", 1 atom = 1000000 uatoms